src/internal/kopia/cleanup_backups.go

```diff
@@ -188,31 +194,57 @@ func cleanupOrphanedData(
 			delete(toDelete, manifest.ID(bm.SnapshotID))
 			delete(toDelete, manifest.ID(ssid))
 
+			// This is a little messy to have, but can simplify the logic below.
+			// The state of tagging in corso isn't all that great right now and we'd
+			// really like to consolidate tags and clean them up. For now, we're
+			// going to copy tags that are related to Reasons for a backup from the
+			// item data snapshot to the backup model. This makes the function
+			// checking if assist backups should be garbage collected a bit easier
+			// because now they only have to source data from backup models.
+			if err := transferTags(d, &bm); err != nil {
+				logger.CtxErr(ctx, err).Errorw(
```

Comment:
I feel minorly uneasy about logging Error level when we don't treat the situation as an error.  Worst case scenario it can lead to debugging confusion where future maintainers will have to decide if an error log was supposed to result in an erroneous run.  The comments help a lot.  But I think an info log would translate a little more nicely here.

Reply:
yea, wasn't really sure what level to put it under. Info makes sense